cmd/agent/main.go

```diff
@@ -257,7 +259,15 @@ func (a *Agent) runHealthServer(o *GrpcProxyAgentOptions) error {
 
 func (a *Agent) runAdminServer(o *GrpcProxyAgentOptions) error {
 	muxHandler := http.NewServeMux()
-	muxHandler.Handle("/metrics", promhttp.Handler())
+	muxHandler.Handle("/metrics", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		host, _, err := net.SplitHostPort(r.Host)
+		// The port number may be omitted if the admin server is running on port 80
```

Comment:
how about "The port number may be omitted if the admin server is running on port 80, the default port for HTTP"?